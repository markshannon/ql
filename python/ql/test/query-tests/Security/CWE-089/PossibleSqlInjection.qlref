Security/CWE-089/PossibleSqlInjection.ql
