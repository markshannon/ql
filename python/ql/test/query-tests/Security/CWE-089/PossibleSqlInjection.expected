edges
| sql_injection.py:9:15:9:21 | django.request.HttpRequest | sql_injection.py:12:8:12:14 | django.request.HttpRequest |
| sql_injection.py:9:15:9:21 | django.request.HttpRequest | sql_injection.py:13:16:13:22 | django.request.HttpRequest |
| sql_injection.py:13:16:13:22 | django.request.HttpRequest | sql_injection.py:13:16:13:27 | django.http.request.QueryDict |
| sql_injection.py:13:16:13:27 | django.http.request.QueryDict | sql_injection.py:13:16:13:39 | externally controlled string |
| sql_injection.py:13:16:13:39 | externally controlled string | sql_injection.py:16:38:16:41 | externally controlled string |
| sql_injection.py:13:16:13:39 | externally controlled string | sql_injection.py:18:39:18:42 | externally controlled string |
| sql_injection.py:13:16:13:39 | externally controlled string | sql_injection.py:21:55:21:58 | externally controlled string |
| sql_injection.py:13:16:13:39 | externally controlled string | sql_injection.py:22:43:22:46 | externally controlled string |
| sql_injection.py:13:16:13:39 | externally controlled string | sql_injection.py:23:45:23:48 | externally controlled string |
| sql_injection.py:18:39:18:42 | externally controlled string | sql_injection.py:18:22:18:42 | externally controlled string |
| sql_injection.py:21:55:21:58 | externally controlled string | sql_injection.py:21:38:21:58 | externally controlled string |
| sql_injection.py:22:43:22:46 | externally controlled string | sql_injection.py:22:26:22:46 | externally controlled string |
| sql_injection.py:23:45:23:48 | externally controlled string | sql_injection.py:23:28:23:48 | externally controlled string |
| sql_injection.py:28:21:28:27 | django.request.HttpRequest | sql_injection.py:30:8:30:14 | django.request.HttpRequest |
| sql_injection.py:28:21:28:27 | django.request.HttpRequest | sql_injection.py:31:16:31:22 | django.request.HttpRequest |
| sql_injection.py:31:16:31:22 | django.request.HttpRequest | sql_injection.py:31:16:31:27 | django.http.request.QueryDict |
| sql_injection.py:31:16:31:27 | django.http.request.QueryDict | sql_injection.py:31:16:31:39 | externally controlled string |
| sql_injection.py:31:16:31:39 | externally controlled string | sql_injection.py:32:66:32:69 | externally controlled string |
| sql_injection.py:32:66:32:69 | externally controlled string | sql_injection.py:32:16:32:69 | externally controlled string |
parents
#select
| sql_injection.py:32:66:32:69 | simple SQL string injection | sql_injection.py:28:21:28:27 | django.request.HttpRequest | sql_injection.py:32:66:32:69 | externally controlled string | This SQL query depends on $@. | sql_injection.py:28:21:28:27 | Django request source | a user-provided value |
